gatsby-node.js

```diff
@@ -57,8 +59,9 @@ const createArticles = async (createPage) => {
   })
 }
 
-const createAutors = async (createPage, slug) => {
+const createAuthors = async (createPage, slug) => {
   const resp = await axios.get(`${BASE_URL}https://www.thedp.com/staff/${slug}.json`)
+  const mostReadDPResp = await axios.get(`${BASE_URL}https://us-central1-web-services-dp.cloudfunctions.net/dropcap/DP`)
```

Comment:
can we destructure data and the properties in data out of data? Also add default params/error handling